instrumentation-genai/opentelemetry-instrumentation-openai-v2/tests/requirements.latest.txt

```diff
@@ -0,0 +1,52 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# ********************************
+# WARNING: NOT HERMETIC !!!!!!!!!!
+# ********************************
+#
+# This "requirements.txt" is installed in conjuction
+# with multiple other dependencies in the top-level "tox.ini"
+# file. In particular, please see:
+#
+#   openai-recent-deps: {[testenv]test_deps}
+#   openai-recent-deps: -r {toxinidir}/instrumentation-genai/opentelemetry-instrumentation-openai-v2/tests/requirements.latest.txt
+#
+# This provides additional dependencies, namely:
+#
+#   opentelemetry-api
+#   opentelemetry-sdk
+#   opentelemetry-semantic-conventions
+#
+# ... with a "dev" version based on the latest distribution.
```

Comment:
Can you instead just move this note to https://github.com/open-telemetry/opentelemetry-python-contrib/blob/main/CONTRIBUTING.md#update-supported-instrumentation-package-versions?

This is actually the norm for most of the packages in this repo (they test against recent versions and opentelemetry-python@HEAD) and there's already some discussion in there.

Reply:
I think it's easier to overlook when the comment is not colocated with the code that it is related to. Putting comments in the affected files rather than centrally, while it can create some redundancy, makes it more obvious.